static/src/js/pages/SearchList/SearchList.jsx

```diff
@@ -216,6 +235,41 @@ const SearchList = ({ limit }) => {
       ]
     }
 
+    if (formattedType === conceptTypes.Visualizations) {
+      return [
+        {
+          className: 'col-auto',
+          dataAccessorFn: buildEllipsisLinkCell,
+          dataKey: 'name',
+          sortFn,
+          title: 'Short Name'
+        },
+        {
+          className: 'col-auto',
+          dataAccessorFn: buildEllipsisTextCell,
+          dataKey: 'title',
+          // SortFn,
```

Comment:
Is there a ticket you can reference so when done, this can be implemented?   Maybe mention the ticket in the comments.

Reply:
Remove comment